src/ui/public/timepicker/pretty_interval.js

```diff
@@ -17,29 +17,39 @@
  * under the License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 const MS_IN_SECOND = 1000;
 const MS_IN_MINUTE = 60 * MS_IN_SECOND;
 const MS_IN_HOUR = 60 * MS_IN_MINUTE;
 const MS_IN_DAY = 24 * MS_IN_HOUR;
 
 export function prettyInterval(intervalInMs) {
+  let interval;
+  let units;
   if (intervalInMs === 0) {
-    return 'Off';
+    return i18n.translate('common.ui.timepicker.off', { defaultMessage: 'Off' });
   } else if (intervalInMs < MS_IN_MINUTE) {
-    const intervalInSeconds = Math.round(intervalInMs / MS_IN_SECOND);
-    const units = intervalInSeconds > 1 ? 'seconds' : 'second';
-    return `${intervalInSeconds} ${units}`;
+    interval = Math.round(intervalInMs / MS_IN_SECOND);
+    units = (interval > 1 ?
```

Comment:
I would suggest we pack this into one i18n string and leverage it's pluralization features, so we can make sure that's working in all languages (also the ones without 2 plural forms):

```js
return i18n.translate('common.ui.timepicker.seconds', {
  defaultMessage: `{interval} {interval, plural,
    one {second}
    other {seconds}
  }`,
  values: { interval },
});
```

And the same for the other units in this file.